tasks/eslint.js

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  grunt.config('eslint', {
+    options: {
+      eslintrc: '.eslintrc',
+      src: ['Gruntfile.js', 'tasks/*.js', 'config/**/*.js', 'node/**/*.js']
+    },
+    app: {
+      options: {eslintrc: 'client/.eslintrc'},
+      src: ['client/*.js', 'client/lib/**/*']
```

Comment:
nit: You can glob this glob to `['client/**/*.js']` and it will do the same thing.
Or not, depends if we want to lint that _client/lib/**_ directory.

Ref: https://github.com/mozilla/fxa-js-client/pull/158/files#r32364753